package.json

```diff
@@ -71,8 +72,8 @@
   "dependencies": {
     "@folio/quick-marc": "^5.0.0",
     "@folio/stripes-acq-components": "^3.0.2",
+    "@folio/stripes-authority-components": "^1.0.0",
```

Comment:
Please move `stripes-authority-components` to `peerDependencies` (reason in [UIPFAUTH-9](https://issues.folio.org/browse/UIPFAUTH-9))